app/controllers/concerns/course/assessment/question/multiple_responses_concern.rb

```diff
@@ -22,6 +22,7 @@ def switch_mcq_mrq_type(multiple_choice, unsubmit)
 
   def unsubmit_submissions
     @question_assessment.assessment.submissions.each do |submission|
+      submission.update('unmark' => 'true') if submission.graded?
```

Comment:
Interesting that this wasn't part of the unsubmission flow before. So in the past, if we unsubmit a graded submission, it would actually remain "marked" in terms of its state?